docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: TraceQL
+menuTitle: TraceQL
+description: Learn about TraceQL, the query language for traces
+weight: 450
+aliases:
+  - /docs/tempo/latest/traceql/
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. A TraceQL query can select traces based on:
+
+- Span and resource attributes, timing, and duration
+- Structural relationships between spans
```

Comment:
Currently TraceQL can do:

- Span and resource attributes, timing, and duration
- Basic aggregates: `count()` and `avg()`

I think the document should be focused on these features. We can add a section for "planned functionality" and include details there.

Reply:
Based upon your comment, shoudl we remove the current bullets and replace them with the two you listed? Are structural relationships between spans and aggregated data from the spans in planned functionality category?